docs/chapters/apps.rst

```diff
@@ -111,7 +111,7 @@ test_value:
   (Optional) If specified, used as the value for this input/parameter when generating the test script.
```

Comment:
Should the word "used" be "use" instead?

Reply:
Re-worded that sentence.

Reply:
Resolving